pyvista/plotting/lookup_table.py

```diff
@@ -513,6 +514,31 @@ def nan_color(self, value):
         self._nan_color_set = True
         self.SetNanColor(*Color(value).float_rgba)
 
+    @property
+    def nan_opacity(self):
+        """Return or set the not a number (NAN) opacity.
+
+        Any values that are NANs will be rendered with this opacity.
+
+        Examples
+        --------
+        Set the NAN opacity to ``0.5``.
+
+        >>> import pyvista as pv
+        >>> lut = pv.LookupTable()
+        >>> lut.nan_color = 'grey'
+        >>> lut.nan_opacity = 0.5
+        >>> lut.plot()
+
+        """
+        color = self.nan_color
+        return color.opacity
+
+    @nan_opacity.setter
+    def nan_opacity(self, value):
+        nan_color = self.nan_color.float_rgba[:3]
+        self.nan_color = Color(nan_color, opacity=value).float_rgba
```

Comment:
```suggestion
        self.nan_color = Color(self.nan_color, opacity=value)
```

Colors can be created from other `Color` instances, so no need to first extract the RGB components yourself. I also removed the conversion to `float_rgba` as I believe `self.nan_color` should be a `Color` instance?